web/src/components/alerting/AlertingPage.tsx

```diff
@@ -1,23 +1,51 @@
 import { PageSection, Title } from '@patternfly/react-core';
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import { HorizontalNav, useActivePerspective } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  HorizontalNav,
+  NamespaceBar,
+  useActivePerspective,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { MonitoringContext, MonitoringProvider } from '../../contexts/MonitoringContext';
 
-const AlertsPage = React.lazy(
-  () => import(/* webpackChunkName: "AlertsPage" */ '../alerting/AlertsPage'),
+const MpCmoAlertsPage = React.lazy(() =>
```

Comment:
we should check if setting the same chunkName does not bloat the chunk including both pages, better to use different chunkNames.